### PR TITLE
Budget simulator smallupdates

### PR DESCRIPTION
**Pull request by bengao01**
Updated the budget simulator with a few small changes, such as switching the order of a few sliders and making the default allocation to be 20% for each category.

### CODE REVIEW

File: frontend/src/budgetSimulation/budgetSimulation.js [hunk 1]
```diff
@@ -33,16 +33,23 @@ class Citizen extends React.Component {
...
-                        (<div key={key}>
-                            <strong>
-                                {trait.split("_").join(" ")}: &nbsp;
-                            </strong>
-                            {this.props.data.traits[trait] ? "True" : "False"}
-                            <br/>
-                        </div>)
+                    Citizen {this.props.data.name} &nbsp;
+                    {Object.keys(this.props.data.traits).map((trait, i) => {
+                        console.log(Object.keys(this.props.data.traits).length);
```
Comment (ryaanahmed) [inline frontend/src/budgetSimulation/budgetSimulation.js]: This block is complicated enough that I'd suggest factoring out as a little helper function, so you end up with something cleaner inside the JSX.

File: frontend/src/budgetSimulation/budgetSimulation.js [hunk 2]
```diff
@@ -33,16 +33,23 @@ class Citizen extends React.Component {
...
+                        return (<>
+                            {this.props.data.traits[trait]
+                                ? trait.split("_").join(" ")
+                                : trait.split("_")
+                                    .join(" ")
+                                    .replace("has", "does not have")
+                                    .replace("is", "is not")
+                            }{i === Object.keys(this.props.data.traits).length - 2
```
Comment (ryaanahmed) [inline frontend/src/budgetSimulation/budgetSimulation.js]: plz no nested ternary! super hard to read.

**Review by ryaanahmed — CHANGES_REQUESTED** [review]
this works well but the block inside the popover is quite difficult to read -- can you factor it out into a little helper function, and get rid of the nested ternaries?

File: frontend/src/budgetSimulation/budgetSimulation.js (hunk 1 above)
Comment (amy-lei) [reply frontend/src/budgetSimulation/budgetSimulation.js]: factored out into a helper function!

File: frontend/src/budgetSimulation/budgetSimulation.js (hunk 2 above)
Comment (amy-lei) [reply frontend/src/budgetSimulation/budgetSimulation.js]: no more nested ternary

File: frontend/src/budgetSimulation/budgetSimulation.js (hunk 1 above)
Comment (mscuthbert) [reply frontend/src/budgetSimulation/budgetSimulation.js]: > factored out into a helper function!

Hearing words like that always makes my heart go a-flutter!

**Comment by ryaanahmed** [comment]
THANK YOU @amy-lei -- much improved.

This made me notice that there are some small problems here that our current linter isn't catching (e.g., it's let through a few `let` vars that should be `const` because they're not being mutated, etc.). I'm going through and adding stricter linting rules on another branch -- as soon as that's merged and merged into this branch, they'll appear as linter failures over here. My fault! But I'll wait to merge this until that's in, so we don't have to cleanup after.

**Comment by ryaanahmed** [comment]
@amy-lei @bengao01 merged latest version of master with the stricter linter in. Please fix the resulting linter errors, then this is good to merge.

**Review by ryaanahmed — APPROVED** [review]
Great! And thanks for the nitpicky cleanup -- this is _way_ more readable now.

**Pull request merged**